debugtools/DDR_VM/src/com/ibm/j9ddr/AuxFieldInfo29.dat

```diff
@@ -148,11 +148,17 @@ J9IdentityHashData.hashData3 = required
 J9IdentityHashData.hashSaltPolicy = required
 J9IdentityHashData.hashSaltTable = required
 J9IndexableObjectContiguous.size = required
+J9IndexableObjectWithDataAddressContiguous.size = required
```

Comment:
This field and `J9IndexableObjectWithDataAddressDiscontiguous.size` cannot be "required"; the have not always existed.